File: work/csv/Range.java
package com.codenjoy.dojo.utils.csv;

public class Range {

    private int min;
    private int max;

    public Range(int min, int max) {
        this.min = min;
        this.max = max;
    }

    public int min() {
        return min;
    }

    public int max() {
        return max;
    }

    public void next() {
        min = max + 1;
        max = min;
    }

    public void min(int value) {
        min = value;
    }

    public void max(int value) {
        max = value;
    }

    public Range copy() {
        return new Range(min, max);
    }

    @Override
    public String toString() {
        return "[" +
                min +
                "-" + max + ']';
    }
}

File: mcp_server/tools/lng_json_to_csv/stuff/range.py
class Range:
    """Python port of Range.java"""
    
    def __init__(self, min_val: int, max_val: int):
        self._min = min_val
        self._max = max_val
    
    def min(self) -> int:
        return self._min
    
    def max(self) -> int:
        return self._max
    
    def next(self):
        self._min = self._max + 1
        self._max = self._min
    
    def set_min(self, value: int):
        self._min = value
    
    def set_max(self, value: int):
        self._max = value
    
    def copy(self):
        return Range(self._min, self._max)
    
    def __str__(self):
        return f"[{self._min}-{self._max}]"
